pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -71,17 +75,25 @@ pub fn sample(ctx: Context<Sample>) -> Result<()> {
 }
 ```
 
-Warning: it is your responsibility to validate that the provided price update is for the appropriate price feed and timestamp.
-`PriceUpdateV2` guarantees that the account contains a verified price for _some_ price feed at _some_ point in time.
-There are various methods on this struct (such as `get_price_no_older_than`) that you can use to implement the necessary checks.
-Note: if you choose the price feed account integration (see below), you can use an account address check to validate the price feed id.
+<Callout type="warning" emoji="⚠️">
+  **NOTE**: Users must validate the price update for the appropriate **price
+  feed** and **timestamp**. `PriceUpdateV2` guarantees that the account contains
+  a verified price for _some_ price feed at _some_ point in time. There are
+  various methods on this struct (such as `get_price_no_older_than`) that users
+  can use to implement the necessary checks.
+</Callout>
+
+<Callout type="info" emoji="ℹ️">
+  **Note**: Use an account address check to validate the price feed ID, if a
+  user wants to integrate using the price feed accounts (see below).
```

Comment:
I like the old text better : 
`if you choose the price feed account integration (see below), you can use an account address check to validate the price feed id.`